### PR TITLE
Add gitlab extension

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/commands/createBranch.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// aha.on('createBranch', async ({ record }) => {
```
Comment (justinpaulson) [inline src/commands/createBranch.ts]: can we remove this file?

File: src/lib/gitlab/validateMrURL.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Validate Pull Request URL
```
Comment (justinpaulson) [inline src/lib/gitlab/validateMrURL.ts]: "Merge" Request

File: src/commands/createBranch.ts (hunk 1 above)
Comment (ghost) [reply src/commands/createBranch.ts]: Ok

**Comment by ghost** [comment]
@justinpaulson  Fixed.

**Pull request merged**